Lib9c.Policy/AccessControlService/IAccessControlService.cs

```diff
@@ -4,6 +4,8 @@ namespace Nekoyume.Blockchain
 {
     public interface IAccessControlService
     {
-        public bool IsAccessDenied(Address address);
+        public bool IsListed(Address address);
```

Comment:
Now, It seems to be unnecessary.

Reply:
Would we ever need to check if an address is in the db? If not, I can remove this too.

Reply:
`GetTxQuota` can handle that functionality.